### DISCUSSION TITLE
How to ensure wires match continously from haptic driver thru RJ45s, Ethernet Cables, etc to tactors

### DISCUSSION THREAD

**Discussion by Neurosmith24** [kriswilk/buzzah#12] [General]
Hi all. I've finally got all of the electronic components to starting putting Buzzah together. I want to use the Ethernet cable and RJ 45 connector idea. But I'm not clear how to make sure each haptic driver's two wires go through the RJ45 connector block, connect to an ethernet cable and then how to find that same signal on the other end of the ethernet connecting to another RJ45 adaptor block and ensure I connect it to the correct Vybronix tactor that it's meant to go to. In other words, how to keep everything matched up properly. 
For ref, here's a crude diagram of what I'm trying to do (based on my understanding PWPInnovator's build). Thanks for any help and insight!
![DIY PD VCR Gloves - PWPInnovator898 (Buzzah Fork)](https://github.com/kriswilk/buzzah/assets/5769522/2a449338-8894-48e4-9eb8-629999f577e3)

**Comment by Neurosmith24** [comment]
Maybe I've figured it out. Looks like RJ45s have a standard pinout - the connectors have 8 wires and are labeled 1 - 8 and color coded. So you just have to make sure haptic driver A wires to pinouts 1 and 2 on the connector (corresponding to green striped and green wires within the ethernet cable). Then on the other end of the ethernet/RJ45 wire the corresponding tactor into pinouts 1 and 2 on that adapter. Is that right?

**Reply by kriswilk** [reply]
Hi @Neurosmith24,

Correct. RJ45 pinout is always numbered in a consistent way and you shouldn't have to concern yourself at all with the cable in between the connectors (as long as it's a regular cable). Whatever you connect to pin N of the first connector should show up on pin N at the other end. That's the beauty of using a standard connector/cable. 👍

**Reply by Neurosmith24** [reply]
Thx!

**Comment by Neurosmith24** [comment]
Somewhat related question... I'm using the 2.54mm 0.1" Pitch 3-pin Jumper Cable that @kriswilk suggested to hook up the haptic driver to RJ45 & ethernet that will run to the LRAs. Question: should I tin the stranded copper wire that's in the jumper cables that inserts into the RJ45 adaptor? Also, should I tin the wires coming from the LRAs that go into the other adaptor? Thx.

**Reply by kriswilk** [reply]
Some will say that the only "proper" way to use a screw terminal adapter is to install a metal ferrule on the end of each wire. This can be important for high-current applications and to ensure the most robust physical connection. But for hobby applications twisting the stranded wire ends and tinning is fine (and far simpler!). There is a chance that the connection may loosen because of the ductility of the solder, but I've never had an issue with it.

**Reply by Neurosmith24** [reply]
Thanks! And now another question (sorry). I had the idea of adding heat shrink to the wires near the connection to the LRA for strain relief and durability. But then I wondered if the heat from the heat gun might damage the LRA. Is that paranoid or a legit concern? Thx.

**Reply by kriswilk** [reply]
The brief heat required to shrink the tubing shouldn't harm the LRAs. If you want to be on the safe side, you can make an aluminum foil shield that you can cover the LRA with temporarily.
